lib/web_of_science/record.rb

```diff
@@ -124,5 +124,29 @@ def find_or_create_model
         rec.database = database
       end
     end
+
+    # Does record have an associated publication? (based on matching PublicationIdentifiers)
+    # Note: must use unique identifiers, don't use ISSN or similar series level identifiers
+    # We search for all PubIDs at once instead of serial queries.  No need to hit the same table multiple times.
+    # @param [WebOfScience::Record] record
```

Comment:
```suggestion
```

i think this `param` comment can go now?

Reply:
yes, good catch -- fixed